service/storage_service.cc

```diff
@@ -840,19 +841,21 @@ class topology_coordinator {
         return {};
     }
 
-    future<> exec_direct_command_helper(raft::server_id id, const raft_topology_cmd& cmd) {
+    future<> exec_direct_command_helper(raft::server_id id, size_t cmd_index, const raft_topology_cmd& cmd) {
         auto ip = _address_map.find(id);
         if (!ip) {
-            slogger.info("raft topology: cannot send command {} to {} because mapping to ip is not available",
-                         cmd.cmd, id);
+            slogger.info("raft topology: cannot send command {} with term {} and index {} "
```

Comment:
Shouldn't that be at least "warn"?

Reply:
maybe, @gleb-cloudius ?

Reply:
Yes, it should higher severity than info. More importantly the id->ip mapping should not be here at all. but that's another story.

Reply:
ok, will update it